src/servers/exporter.ts

```diff
@@ -69,6 +70,7 @@ const business_g = new client.Gauge({
 })
 
 const roles = ["dealer", "funder", "bankowner"]
+const account_roles = [getDealerWalletId(), getFunderWalletId(), getBankOwnerWalletId()]
```

Comment:
camlCase

Reply:
?

Reply:
`account_roles ` => `accountRoles` - or is it `snake_case` because a dependency expects it?

Reply:
oh yes ok. was looking at the array itself: getDealerWalletId and co.